pkg/blockstorage/helpers.go

```diff
@@ -33,8 +33,11 @@ const (
 	AzureCientID               = "AZURE_CLIENT_ID"
 	AzureClentSecret           = "AZURE_CLIENT_SECRET"
 	AzureResurceGroup          = "AZURE_RESOURCE_GROUP"
+	AzureResurceMgrEndpoint    = "AZURE_RESOURCE_MANAGER_ENDPOINT"
 	AzureMigrateStorageAccount = "AZURE_MIGRATE_STORAGE_ACCOUNT_NAME"
 	AzureMigrateStorageKey     = "AZURE_MIGRATE_STORAGE_ACCOUNT_KEY"
+	AzureActiveDirEndpoint     = "AZURE_AD_ENDPOINT"
```

Comment:
Update this test with the new configs ? 
https://github.com/kanisterio/kanister/blob/ab6010860929c1702fb94c238607a6115bcc621a/pkg/blockstorage/blockstorage_test.go#L301

Reply:
Discussed this with @PrasadG193  -  He will address this in a follow up PR.